docs/projects/led-guide.md

```diff
@@ -0,0 +1,524 @@
+# Creating your first node
+
+## Repo Structure
+
+At first the repository will seem a bit intimidating! There's plenty
+going on here, but for the purposes of this guide, you don't need to
+worry about most of it.
+
+As a prequiste, make sure you've read through Jacob's
+[getting started guide](https://github.com/CooperUnion/selfdrive#getting-started)
+on setting up the repository.
+
+We'll be starting off in the `selfdrive/dbw/node_fw/mod` directory the
+directory, as shown below.
+
+```bash
+mod
+├── blink
+│   ├── blink.c
+│   ├── blink.h
+│   └── library.json
+├── brake
+│   ├── brake.c
+│   ├── brake.h
+│   └── library.json
+├── ctrl
+│   ├── ctrl.c
+│   ├── ctrl.h
+│   └── library.json
+├── encoder
+│   ├── encoder.c
+│   ├── encoder.h
+│   └── library.json
+├── pb_mon
+│   ├── library.json
+│   ├── pb_mon.c
+│   └── pb_mon.h
+├── steer
+│   ├── library.json
+│   ├── steer.c
+│   └── steer.h
+├── sup
+│   ├── library.json
+│   ├── sup.c
+│   └── sup.h
+└── throttle
+    ├── library.json
+    ├── pedal.c
+    ├── pedal.h
+    ├── throttle.c
+    └── throttle.h
+```
+
+Within this directory are all the `.c` and `.h` programs for each board.
+This is where we'll be writing the firmware for our program!
+
+## Creating `led.c` and `led.h`
+
+Create a new `led` directory and within it, add `led.c`, `led.h` and a
```

Comment:
perhaps include the contents of led.h? 👨🏽‍🦲 

#ifndef LED_H
#define LED_H

#endif